app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_bench/AddBenchStatusOnBusStop.kt

```diff
@@ -0,0 +1,44 @@
+package de.westnordost.streetcomplete.quests.bus_stop_bench
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.osmquest.SimpleOverpassQuestType
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.osm.mapdata.OverpassMapDataAndGeometryApi
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment
+import de.westnordost.streetcomplete.settings.ResurveyIntervalsStore
+
+class AddBenchStatusOnBusStop(o: OverpassMapDataAndGeometryApi, r: ResurveyIntervalsStore) : SimpleOverpassQuestType<Boolean>(o) {
+
+    override val tagFilters = """
+        nodes with
+        (
+          (public_transport = platform and ~bus|trolleybus|tram ~ yes)
+          or
+          (highway = bus_stop and public_transport != stop_position)
+        )
+        and physically_present != no and naptan:BusStopType != HAR
```

Comment:
Ah now I see why you were asking in the other PR. Is it worth pulling this out so it's a common rule across these two bus stop quests and any future ones. Or at least adding comments to both saying to update the other one if it changes.

Or actually TBH all of this filter apart from the bench bit could be common couldn't it?